tests/e2e/containers/config.go

```diff
@@ -24,10 +24,10 @@ const (
 	// It should be uploaded to Docker Hub. OSMOSIS_E2E_SKIP_UPGRADE should be unset
 	// for this functionality to be used.
 	previousVersionOsmoRepository = "osmolabs/osmosis"
-	previousVersionOsmoTag        = "22.0.0-alpine"
+	previousVersionOsmoTag        = "23.0.0-rc1-alpine"
 	// Pre-upgrade repo/tag for osmosis initialization (this should be one version below upgradeVersion)
 	previousVersionInitRepository = "osmolabs/osmosis-e2e-init-chain"
-	previousVersionInitTag        = "22.0.0"
+	previousVersionInitTag        = "23.0.0-rc1-temp"
```

Comment:
Tagged image from https://github.com/osmosis-labs/osmosis/pull/7473

Once that PR is merged, the v23 release will be good and we can use that for both of these tags instead.